packages/ra-core/src/controller/details/useCreateController.ts

```diff
@@ -118,6 +115,11 @@ export const useCreateController = <
     const recordToUse = getRecord(location, record);
     const version = useVersion();
 
+    const { resource } = useResource(props);
```

Comment:
I think the controllers should only pass the resource down for BC. since the context is already there, components down the tree can already get it